cg/meta/transfer/external_data.py

```diff
@@ -135,6 +127,16 @@ def check_fastq_md5sum(self, fastq_path) -> Optional[Path]:
             if not check_md5sum(file_path=fastq_path, md5sum=given_md5sum):
                 return fastq_path
 
+    def get_available_samples(self, folder: Path, ticket_id: int) -> List[models.Sample]:
+        """Returns the samples from given ticket that are present in the provided folder"""
+        available_folders: List[str] = [sample_path.parts[-1] for sample_path in folder.iterdir()]
+        transferred_samples: List[models.Sample] = [
```

Comment:
```suggestion
        available_samples: List[models.Sample] = [
```
They have not been transferred yet right?